.evergreen/config.yml

```diff
@@ -1276,6 +1306,24 @@ tasks:
     commands:
     - func: "run oidc auth test with test credentials"
 
+  - name: "oidc-auth-test-azure-latest"
+    commands:
+    - command: shell.exec
```

Comment:
We're trying to move away from `shell.exec` and the corresponding chunks of shell code mixed in with the config - can this be pulled out to its own `.sh` file and switched to use `subprocess.exec`?

Reply:
It has issues finding the proper directories on the VM. The python driver is using shell.exec also, probably for this reason.